staker/staker.go

```diff
@@ -486,6 +643,58 @@ func (s *Staker) Start(ctxIn context.Context) {
 		}
 		return s.config.StakerInterval
 	})
+	s.CallIteratively(func(ctx context.Context) time.Duration {
```

Comment:
this is already done in the first CallIteratively from Start. No need to do it twice